themes/mlibrary_new/simple-pages/page/about.php

```diff
@@ -0,0 +1,8 @@
+<?php 
+$text = metadata('simple_pages_page', 'text', array('no_escape' => true));
+if (metadata('simple_pages_page', 'use_tiny_mce')) {
+   echo $text;
+} else {
+   echo eval('?>' . $text);
```

Comment:
Why is this an eval?  That's a really risky proposition.

Reply:
I used the escape no need for this.